packages/v4/src/content/design-guidelines/demos/primary-detail/primary-detail.md

```diff
@@ -67,4 +67,12 @@ Here are some other best practices to keep in mind:
 - If the details pane can be toggled close, there needs to be a close icon in the top-right of the details pane. Users may also toggle open/close in a toolbar.
  
 For [empty states](/components/empty-state), if the details pane is automatically expanded on default but a primary item is not selected by default, then there should be an empty state in the details pane directing users to make a selection in the primary. In cases where there is bulk select capability in the primary component, an expanded details drawer should display an empty state informing users that multiple items have been selected. To view details of a single primary item, the user must clear their selection.
- 
+
+## Splitter in a primary-detail
+
+A **splitter** allows you to create a layout with resizable panes. The orientation of a splitter can be set to **vertical** or **horizontal**.
+
+### Usage
+Add a splitter to a primary-detail if you need to resize the width or height of a panel and give more space to a given content. It helps to prioritize the content you’re focusing on. A splitter resizes the width or height of a panel depending on the orientation of a splitter. If data shown in a primary-detail has enough space, then you don’t need to use a splitter.
```

Comment:
Same thing here with re-wording.
Change to " Add a splitter to a primary-detail if you need to resize the width or height of a panel to give content more space", and delete "A splitter resizes the width or height of a panel depending on the orientation of a splitter."